shimmy/__init__.py

```diff
@@ -73,7 +73,7 @@ def __call__(self, *args: list[Any], **kwargs: Any):
 ]
 
 
-__version__ = "0.2.1"
+__version__ = "1.0.0"
```

Comment:
I didn't tell another about this, but Gymnasium has shimmy `<1.0` https://github.com/Farama-Foundation/Gymnasium/blob/30e846aec5cd510f36e8f1b1249fb35f2e32d924/pyproject.toml#L39
As I intended that in shimmy 1.0 then atari would be removed as we shouldn't be providing atari support if it updates to Gymnasium. 

Also we don't need to change the version number to make shimmy mature therefore we need to revert this change at least.